pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -704,19 +708,14 @@ func TestGenericScheduler(t *testing.T) {
 				Framework: fwk,
 			}
 
-			var pvcs []v1.PersistentVolumeClaim
-			pvcs = append(pvcs, test.pvcs...)
-			pvcLister := fakeframework.PersistentVolumeClaimLister(pvcs)
-
 			scheduler := NewGenericScheduler(
 				cache,
 				snapshot,
 				[]framework.Extender{},
-				pvcLister,
 				false,
 				schedulerapi.DefaultPercentageOfNodesToScore)
 			result, err := scheduler.Schedule(context.Background(), prof, framework.NewCycleState(), test.pod)
-			if !reflect.DeepEqual(err, test.wErr) {
+			if err != test.wErr && err.Error() != test.wErr.Error() {
```

Comment:
off the topic though, but this change is nice!
this fix the warning in my IDE:
```avoid using reflect.DeepEqual with errorsdeepequalerrors```

maybe we can do similar thing there,
https://github.com/kubernetes/kubernetes/blob/c46c1c043ee8de4c021c03ddb0fb157d1c1ba3f8/pkg/scheduler/scheduler_test.go#L345